e2e_tests/tests/cluster/test_users.py

```diff
@@ -181,6 +182,22 @@ def extract_id_and_owner_from_exp_list(output: str) -> List[Tuple[int, str]]:
     return [(int(r[0]), r[1]) for r in rows]
 
 
+@pytest.mark.e2e_cpu
+def test_post_user_api(clean_auth: None) -> None:
+    master_url = conf.make_master_url()
+    authentication.cli_auth = authentication.Authentication(
+        conf.make_master_url(), requested_user="admin", password="", try_reauth=True
+    )
+    new_username = get_random_string()
+
+    user = bindings.v1User(active=True, admin=False, username=new_username, id=0)
```

Comment:
I like your solution.

Reply:
Do you know if there's a good way to put the commit on this PR? I've edited individual files in the web UI before but I don't know for a whole commit. Otherwise I'll make a new PR with just that change.

Reply:
I don't think there's a way to commandeer a PR unless it's an upstream branch, so yes probably create a new one.